baybe/kernels/basic.py

```diff
@@ -43,3 +44,24 @@ class MaternKernel(Kernel):
 
     Only takes the values 0.5, 1.5 or 2.5. Larger values yield smoother interpolations.
     """
+
+    lengthscale_prior: Optional[Prior] = field(default=None)
+    """An optional prior on the kernel lengthscale."""
+
+    def to_gpytorch(self, *args, **kwargs):  # noqa: D102
+        # See base class.
+        # Provide the transformed lengthscale as keyword argument.
+        return super().to_gpytorch(
+            lengthscale_prior=self.lengthscale_prior.to_gpytorch()
+        )
+
+
+@define(frozen=True)
+class ScaleKernel(Kernel):
+    """A kernel for decorating existing kernels with an outputscale."""
+
+    base_kernel: Kernel = field(factory=MaternKernel)
```

Comment:
I don't think we should provide a default here. You'll always have a specific base kernel in mind when using a scale kernel. And there is no way someone could reasonably assume that a MaternKernel is chosen by default without looking into the code (it's a very random choice), so nothing is gained except perhaps unintended effects.

```suggestion
    base_kernel: Kernel = field()
```